src/utils/generateMessage.js

```diff
@@ -36,17 +36,16 @@ export default function({
               response,
               voice,
             }).then(susiMessages => {
-              susiMessages.forEach(eachMessage => {
-                if (eachMessage.planDelay) {
-                  tasks.push(
-                    plannedFunction,
-                    () => createSusiMessage({ message: eachMessage }),
-                    eachMessage.planDelay,
-                  );
-                } else {
-                  createSusiMessage({ message: eachMessage });
-                }
-              });
+              if (susiMessages[0].planDelay) {
```

Comment:
This is the incorrect fix, we need all the messages that comes in the `susiMessages` array. This piece of code was added to make Planned actions work.

Reply:
ok @akshatnitd I will fix it removing duplicate entries from the `susiMessages` array and then runnnig `forEach` loop over the array so as to print all the messages. Can you suggest if this approach is correct?

Reply:
Does `susiMessages` contain duplicate entries? Can you give an example case?

Reply:
@akshatnitd when I have query : `who is mario` , I get following list of actions(as observed in networking tab of console):-
`
"actions": [
      {
        "language": "en",
        "type": "answer",
        "expression": "Great question. I never actually thought about that before, and I am not sure."
      },
      {
        "type": "answer",
        "mood": "sabta",
        "expression": "Great question. I never actually thought about that before, and I am not sure."
      }
    ],
`

Reply:
@akshatnitd changes has been done so as to include non-duplicate, multiple messages

Reply:
@Vineet-Sharma20 For this specifically, this should be fixed from backend and only 1 action should be ideally coming from backend.

Reply:
@akshatnitd ok

Reply:
@akshatnitd I have opened this as an issue in susi_server

Reply:
@Vineet-Sharma29 Thanks! I think you can close this PR as it is a server issue!